### PR TITLE
Dev 1.3.2 clear entrance

### PR DESCRIPTION
**Pull request by huangKai-2323**
### What is the purpose of the change

EngineConn-Core defines the the abstractions and interfaces of the EngineConn core functions.
The Engine Service in Linkis 0.x is refactored, EngineConn will handle the engine connection 
and session management.

### Related issues/PRs

Related issues: #590
Related pr:#591

### Brief change log

- Define the core abstraction and interfaces of the EngineConn Factory;
- Define the core abstraction and interfaces of Executor Manager.

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: docs/info-1.3.2.md [hunk 1]
```diff
@@ -6,3 +6,4 @@
 | linkis | 新增  | wds.linkis.governance.station.admin.token |   /具有管理员权限的特殊token|
 
 
+| cg-entrance | 新增  | wds.linkis.entrance.whether.open.send.instance | true |取值范围：true或false|
```
Comment (casionone) [inline docs/info-1.3.2.md:9]: key 修改为 wds.linkis.entrance.auto.check.job.status.enable

File: linkis-public-enhancements/linkis-jobhistory/src/main/scala/org/apache/linkis/jobhistory/service/impl/JobHistoryQueryServiceImpl.scala [hunk 2]
```diff
@@ -428,4 +431,28 @@ class JobHistoryQueryServiceImpl extends JobHistoryQueryService with Logging {
...
+    statusList.add(TaskStatus.Running.toString)
+    val eDate = new Date(System.currentTimeMillis)
+    val sDate = DateUtils.addDays(eDate, -1)
+    val jobHistoryList =
+      jobHistoryMapper.search(null, null, statusList, sDate, eDate, null, null, request.instance)
+    val idlist = jobHistoryList.asScala.map(_.getId).asJava
+    logger.info("Tasks id will be canceled id :{}", idlist)
+    // 修改task状态
+    if (!idlist.isEmpty) {
+      jobHistoryMapper.updateJobHistoryCancelById(idlist)
```
Comment (casionone) [inline linkis-public-enhancements/linkis-jobhistory/src/main/scala/org/apache/linkis/jobhistory/service/impl/JobHistoryQueryServiceImpl.scala:454]: 是否考录也更新下 error_msg 字段 
msg为 “entrance service restart, automatically cancel tasks that have not been completed for a long time (24h)(entrance 服务重启，自动将长时间（24h）未完成的任务置为取消状态)

File: linkis-public-enhancements/linkis-jobhistory/src/main/scala/org/apache/linkis/jobhistory/service/impl/JobHistoryQueryServiceImpl.scala [hunk 3]
```diff
@@ -428,4 +431,28 @@ class JobHistoryQueryServiceImpl extends JobHistoryQueryService with Logging {
     jobHistoryMapper.updateOberverById(jobHistory.getId, jobHistory.getObserveInfo)
   }
 
+  @Receiver
+  override def clearUndoneTasksByEntranceInstance(
+      request: EntranceInstanceConfRequest,
+      sender: Sender
+  ): Unit = {
+    // 查询未完成的task
```
Comment (casionone) [inline linkis-public-enhancements/linkis-jobhistory/src/main/scala/org/apache/linkis/jobhistory/service/impl/JobHistoryQueryServiceImpl.scala:439]: 用英文

File: linkis-public-enhancements/linkis-jobhistory/src/main/resources/mapper/common/JobHistoryMapper.xml [hunk 4]
```diff
@@ -221,4 +221,12 @@
     <update id="updateOberverById">
         update linkis_ps_job_history_group_history set observe_info = #{observeInfo} where id = #{taskid}
     </update>
+
+    <update id="updateJobHistoryCancelById" >
+        update linkis_ps_job_history_group_history set status = 'Cancelled'
+        WHERE id IN
+        <foreach collection="idList" item="id" separator="," open="(" close=")">
+            #{id}
+        </foreach>
```
Comment (casionone) [inline linkis-public-enhancements/linkis-jobhistory/src/main/resources/mapper/common/JobHistoryMapper.xml:230]: 考虑加一下 数量 limit  限制

File: docs/info-1.3.2.md (hunk 1 above)
Comment (huangKai-2323) [reply docs/info-1.3.2.md:9]: done

File: linkis-public-enhancements/linkis-jobhistory/src/main/scala/org/apache/linkis/jobhistory/service/impl/JobHistoryQueryServiceImpl.scala (hunk 3 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-jobhistory/src/main/scala/org/apache/linkis/jobhistory/service/impl/JobHistoryQueryServiceImpl.scala:439]: done

File: linkis-public-enhancements/linkis-jobhistory/src/main/resources/mapper/common/JobHistoryMapper.xml (hunk 4 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-jobhistory/src/main/resources/mapper/common/JobHistoryMapper.xml:230]: done

File: linkis-public-enhancements/linkis-jobhistory/src/main/scala/org/apache/linkis/jobhistory/service/impl/JobHistoryQueryServiceImpl.scala (hunk 2 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-jobhistory/src/main/scala/org/apache/linkis/jobhistory/service/impl/JobHistoryQueryServiceImpl.scala:454]: done

**Review by casionone — APPROVED** [review]

**Pull request merged**